tests/test_plugin.py

```diff
@@ -13,55 +14,52 @@
 from .utils import disable_versioning_for_navigation
 
 
-class NavigationSelectorTestCase(TestCase):
+try:
+    from djangocms_versioning.constants import ARCHIVED, DRAFT, UNPUBLISHED, PUBLISHED
+except ImportError:
+    ARCHIVED, DRAFT, PUBLISHED, UNPUBLISHED = None
```

Comment:
This doesn't look right to me... I don't think you should do try/except ImportError in tests as this could have unexpected consequences (like say a test might end up checking that something is equal to None instead of 'archived').
But also, I don't think you actually use this import anywhere?

Reply:
Sorry, this must be left while I was trying something...removed.